snorkel/mtl/loggers/log_manager.py

```diff
@@ -114,9 +86,9 @@ def trigger_evaluation(self) -> bool:
 
     def trigger_checkpointing(self) -> bool:
         """Check if triggers the checkpointing"""
-        if not self.checkpointing:
+        if self.checkpointer is None:
             return False
-        satisfied = self.trigger_count >= self.checkpointing_freq
+        satisfied = self.trigger_count >= self.checkpointer.checkpoint_factor
```

Comment:
Why is this changing now?

Reply:
Previously log_manager pulled out `checkpoint_factor` and called it `checkpoint_freq` (a mistake). Now it just pulls the value directly from the checkpointer. So the value is the same as before, it's just being called the right thing now.